src/components/Recorder.js

```diff
@@ -92,6 +92,8 @@ export default class Recorder extends H5P.EventDispatcher {
 
     const loadAudioUrl = new Promise((resolve, reject) => {
       this.once('wav-delivered', e => {
+        console.log('Uploading audio to the server...')
+        this._uploadAudioToServer(e.data);
```

Comment:
Instead of calling this `_uploadAudioToServer`, call it what it is - something like `_dispatch_audio_as_event`.

Reply:
All of the function names are using `camelCase` instead of `snake_case` in this code base. Did you mean `_dispatchAudioAsEvent`?